txcache/virtualSelectionSession.go

```diff
@@ -57,6 +59,39 @@ func (virtualSession *virtualSelectionSession) getNonce(address []byte) (uint64,
 	return account.initialNonce.Value, nil
 }
 
+func (virtualSession *virtualSelectionSession) accumulateConsumedBalance(tx *WrappedTransaction) error {
+	sender := tx.Tx.GetSndAddr()
+	feePayer := tx.FeePayer
+
+	senderRecord, err := virtualSession.getRecord(sender)
+	if err != nil {
+		log.Warn("accumulateConsumedBalance.getRecord sender",
```

Comment:
Maybe "virtualSelectionSession.accumulateConsumedBalance(), err when getting record for sender" (opionion; below as well).